src/content/docs/distributed-tracing/infinite-tracing/set-trace-observer.mdx

```diff
@@ -14,22 +14,164 @@ redirects:
   - /docs/understand-dependencies/distributed-tracing/infinite-tracing/set-trace-observer
 ---
 
-If you're following the [Enable distributed tracing](/docs/understand-dependencies/distributed-tracing/enable-configure/overview-enable-distributed-tracing) procedures and want to set up Infinite Tracing, you'll need to set up a trace observer.
+If you are following the [steps](/docs/distributed-tracing/enable-configure/quick-start) to set up distributed tracing and are continuing with the steps for the advanced option called Infinite Tracing, you need to set up a trace observer. The trace observer is what allows you to go beyond the basic [head-based sampling](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) of standard distributed tracing. 
 
-A trace observer is a cloud-based New Relic tool that decides what trace data to keep and send to New Relic. The trace observer lives in New Relic Edge, which is an AWS-based service that provides you with a low-latency and low-cost way to send your telemetry data to New Relic. The trace observer sends data via our [Trace API](/docs/understand-dependencies/distributed-tracing/trace-api/introduction-trace-api), which is the entry point for all distributed trace data we ingest.
+The trace observer is a cloud-based New Relic tool that uses [tail-based sampling](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#tail-based) to decide what trace data to keep and send to New Relic. The trace observer is an AWS-based service that provides you with a low-latency and low-cost way to send your telemetry data to New Relic. The trace observer sends data via our Trace API, which is the entry point for all distributed trace data we ingest.
```

Comment:
I think it might help to make it clear that the trace observer is what allows Infinite Tracing to work. As it is, the two concepts don't seem exactly aligned, so one might be left with the impression that the trace observer is something separate or not directly related to Infinite Tracing. If that makes sense. So maybe something like 'The trace observer is the tool that implements Infinite Tracing's tail-based sampling. The trace observer is...'

Reply:
@zuluecho9, I modified a sentence in the previous paragraph: The trace observer is a component of Infinite Tracing that allows you to go beyond the basic [head-based sampling](/docs/distributed-tracing/concepts/how-new-relic-distributed-tracing-works/#head-based) of standard distributed tracing.